syz-verifier/stats.go

```diff
@@ -16,12 +16,13 @@ import (
 // of the verification process.
 type Stats struct {
 	// Calls stores statistics for all supported system calls.
-	Calls            map[string]*CallStats
-	TotalMismatches  int64
-	TotalProgs       int64
-	FlakyProgs       int64
-	MismatchingProgs int64
-	StartTime        time.Time
+	Calls               map[string]*CallStats
+	TotalCallMismatches int64
```

Comment:
Not related to the current change, but is it ok that you update it via `atomic.Add***`, but read directly instead of `atomic.Load***`?

Reply:
Don't think it is Ok. Thank you.